app/loc/lang-sv.json

```diff
@@ -1,28 +1,28 @@
 {
-    "STR_CONFIRM_CLOSE_APP": "Är du säker att du vill stänga?",
-    "STR_APPNAME": "Luottamushenkilön työpöytä",
+    "STR_CONFIRM_CLOSE_APP": "Är du säker på att du vill stänga skrivbordsapplikationen?",
+    "STR_APPNAME": "Förtroendevaldas skrivbord",
     "STR_LOGIN": "Logga in",
-    "STR_EXPANDCOLLAPSE": "Expandera / Minimize",
+    "STR_EXPANDCOLLAPSE": "Expandera / Minimera",
     "STR_LOGOUT": "Logga ut",
-    "STR_MTG_LOGIN": "Logga in",
-    "STR_MTG_LOGIN_PROGRESS": "Sökar sammanträdet",
+    "STR_MTG_LOGIN": "Logga in till sammanträdet",
+    "STR_MTG_LOGIN_PROGRESS": "Söker sammanträdet",
     "STR_MTG_EXIT_PROGRESS": "Lämnar sammanträdet",
-    "STR_MTG_LOGOUT": "Lämna sammanträdet",
-    "STR_MTG_LOGOUT_CONFIRM": "Är du säker du vill stänga sammanträdet?",
+    "STR_MTG_LOGOUT": "Logga ut ur sammanträdet",
+    "STR_MTG_LOGOUT_CONFIRM": "Är du säker på att du vill lämna sammanträdet?",
```

Comment:
The translation is now "Är du säker på att du vill lämna sammanträdet?" "stänga" is not used.